packages/functional-tests/pages/login.ts

```diff
@@ -497,8 +497,14 @@ export class LoginPage extends BaseLayout {
   }
 
   async setNewPassword(password: string) {
-    await this.page.locator(selectors.PASSWORD).fill(password);
-    await this.page.locator(selectors.VPASSWORD).fill(password);
+    const config = await this.getConfig();
+    if (config.showReactApp.resetPasswordRoutes === true) {
```

Comment:
Lets file a follow up to rip out all the old password reset flows and update the tests.

Reply:
(Same as other comment) Filed FXA-8267